lib/vsc/utils/rest.py

```diff
@@ -215,6 +215,8 @@ def get_connection(self, method, url, body, headers):
             sep = '/'
         else:
             sep = ''
+        if self.url.endswith('/') and url.startswith('/'):
```

Comment:
what url are you trying to parse here?

Reply:
@matesaki i'm mainly puzzled why the code above does work.
also add a unittest for this

Reply:
Condition above check if there is no slash before merging. If true, add one.
This new condition check if there are to many slashes. If true, remove one.

It may be confusing because there are two variables url, first Ends with '/' and second Starts with '/'.

Trying to make a test ...

Reply:
don't know how to test it

Reply:
i think the whole code block, incl the previous lines can become
```
url = url.lstrip('/')
if self.url.endswith('/'):
    sep = '/'
else:
    sep = ''
```

Reply:
yes, this is equivalent
(rebased)